app/src/main/java/openfoodfacts/github/scrachx/openfood/views/MainActivity.java

```diff
@@ -115,6 +147,13 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         setContentView(R.layout.activity_main);
 
+        final PeriodicWorkRequest periodicWorkRequest = new PeriodicWorkRequest.Builder(OfflineEditPendingProductsWorker.class, 60, TimeUnit.MINUTES, 10, TimeUnit.MINUTES)
+            .setConstraints(new Constraints.Builder().setRequiredNetworkType(NetworkType.CONNECTED).build())
+            .addTag(OFFLINE_EDIT_PENDING_WORK_NAME)
+            .build();
+
+        WorkManager.getInstance(getApplicationContext()).enqueueUniquePeriodicWork(OFFLINE_EDIT_PENDING_WORK_NAME, ExistingPeriodicWorkPolicy.KEEP, periodicWorkRequest);
+
```

Comment:
Is it possible to use ExistingPeriodicWorkPolicy.REPLACE instead of ExistingPeriodicWorkPolicy.KEEP.
The MainActivity can recreated: I added a breakpoint, use the application and noticed that the breakpoint can be hitted several times. So I guess that the work can be launched several times.

Reply:
Yes, the breakpoint must have hit several times but the work would only be created once since I have used [`enqueueUniquePeriodicWork`](https://developer.android.com/reference/androidx/work/WorkManager#enqueueUniquePeriodicWork(java.lang.String,%20androidx.work.ExistingPeriodicWorkPolicy,%20androidx.work.PeriodicWorkRequest)). Also the policy is kept as KEEP so that whenever this line is executed it doesn't replace the old work so that the timer is not refreshed again. Reference for [Policies](https://developer.android.com/reference/androidx/work/ExistingWorkPolicy.html).